pycbc/psd/variation.py

```diff
@@ -74,11 +80,15 @@ def calc_psd_variation(strain, psd_short_segment, psd_long_segment,
         kmin = int(low_freq / psd_long.delta_f)
         kmax = int(high_freq / psd_long.delta_f)
         # Comapre the PSD of the short segment to the long segment
-        #diff = numpy.array([numpy.std((p_short[kmin:kmax] /
-        #           psd_long[kmin:kmax])) for p_short in psd_short])
-        diff = numpy.array([(p_short[kmin:kmax] / psd_long[kmin:kmax]).sum()
-                           for p_short in psd_short])
-        diff /= (kmax - kmin)
+        freqs = FrequencySeries(psd_long.sample_frequencies,
+                                delta_f = psd_long.delta_f,
+                                epoch = psd_long.epoch, dtype = fs_dtype)
+        weight = numpy.array(
```

Comment:
I suggest adding an inline comment here about what is being done with this weighting and why. 
Then the sum(step) is integrating a weighted ratio of PSDs over frequency, correct?

Reply:
yep that's right. I've added in a comment so it's more obvious